l10n_ch_scan_bvr/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Scan ESR/BVR to create supplier invoices
+========================================
+
+This module works with C-channel or other OCR scanner.
+
+It helps you to create an invoice directly from the ESR/BVR Code.
+Find the menu entry called "Scan BVR" under Accounting -> Supplier.
+It opens a popup from which you can scan the ESR/BVR number.
+It'll recognize the needed information and create an
+invoice for the right supplier.
+
+If you have completed the field "Default product supplier invoice"
+on the concerned supplier,
+it'll create a line with the proper amount and the given product.
+
+It currently supports BVR and BVR+
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Optionnally define a product in any supplier's form you may use
+* Open the wizard: " Accounting > BVR Scan"
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/125/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+l10n-switzerland/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+l10n-switzerland/issues/new?body=module:%20
+l10n_ch_scan_bvr%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Alex Comba <alex.comba@agielbg.com>
```

Comment:
@yvaucher, I didn't know during the module migration I had to watch the commit history in the previous version. 
Anyway, I have no problem to add them as contributors :wink: